types/vis/vis-tests.ts

```diff
@@ -77,7 +80,7 @@ console.log('formatted items', items);
 data = new vis.DataSet<TestData>();
 
 // subscribe to any change in the DataSet
-data.on('*', (event, properties, senderId) => {
+data.on('*', (event: any, properties: any, senderId: any) => {
```

Comment:
do not use any

Reply:
This in fact is not related with this pull request, but hooks (which have rules newer than latest vis change) are failing because implicit type warning. This is just to make hooks happy, I really dont think this should be a blocker for this PR because in fact i dont know what should be there.